debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerQueryResultProcessor.java

```diff
@@ -156,6 +157,11 @@ int processResult(ResultSet resultSet) {
 
             // DML
             if (operationCode == RowMapper.INSERT || operationCode == RowMapper.DELETE || operationCode == RowMapper.UPDATE) {
+                final TableId tableId = RowMapper.getTableId(connectorConfig.getCatalogName(), resultSet);
```

Comment:
So we are getting the entries for _all_ tables and only filter out later on, correct? Did you consider crafting a WHERE class based on the filter configuration, so to select only the captured schemas (I think then we'd effectively be doing the same as before)?

Reply:
Or maybe not, given the changes further below? Why do we do this filtering after the fact here?

Reply:
This is to handle a situation where the configuration does not apply any include/exclude lists but we need to still filter out the events we process for objects only in the configured `database.pdb.name` catalog.  I don't believe there was a way to do this directly with the SQL query, but I can take one last pass to see if we can.

Reply:
So after reviewing the code, the missing link was `SystemTablesPredicate`.  This filter is used in order for the programmatic code to exclude certain tables that are in known excluded schemas such as `SYS`.  The dynamic nature of the LogMiner query was not taking into account these "excluded" schemas.

For now, I'll move these tables defined in `SystemTablesPredicate` likely to a `SystemTables` class that both the predicate and the `SqlUtils` class used by LogMiner can be based on.  This would allow us to maintain this list once and should solve the concern with the additional filter call you referenced above.  Once I've made the changes & verified the tests pass, I'll push the rebase & this last change.